src/runtime/manifest-primitive-parser/smoke/toy.html

```diff
@@ -0,0 +1,161 @@
+<style>
```

Comment:
Nice!

I'd suggest a `tools` dir instead of `smoke`, and maybe a different filename; playground.html, manifest-editor.html, ast-inspector.html?

It should also have instructions on how to use it (i.e. run a server at the project top level and open `http://localhost:<port>/src/runtime/manifest-primitive-parser/smoke/toy.html`).

Reply:
Ok. Fwiw, it should work opened via any server (other than file://), doesn't require 'top level' anything.